--- lib/gis/CMakeLists.txt.orig	2026-01-31 11:50:06.388902846 -0700
+++ lib/gis/CMakeLists.txt	2026-01-31 11:43:28.154668910 -0700
@@ -7,8 +7,8 @@
 set(grass_gis_DEFS "-DGRASS_VERSION_DATE=\"${GRASS_VERSION_DATE}\"")
 if(MSVC)
   set(grass_gis_DEFS "${grass_gis_DEFS};-D_USE_MATH_DEFINES=1")
-  set(gislib_INCLUDES "../../msvc")
-  list(APPEND gislib_SRCS "../../msvc/dirent.c;../../msvc/fcntl.c")
+  set(gislib_INCLUDES "${PROJECT_SOURCE_DIR}/msvc")
+  list(APPEND gislib_SRCS "${PROJECT_SOURCE_DIR}/msvc/dirent.c;${PROJECT_SOURCE_DIR}/msvc/fcntl.c")
 endif()
 
 build_module(
