--- lib/init/CMakeLists.txt.orig	2026-02-28 12:44:59.279405329 -0700
+++ lib/init/CMakeLists.txt	2026-04-28 11:05:38.255399605 -0600
@@ -50,17 +50,29 @@
 
 # configure and install grass.py
 set(GRASS_PYDIR ${GRASS_INSTALL_PYDIR_RUNTIME_PATH})
-configure_file(grass.py ${OUTDIR}/${CMAKE_INSTALL_BINDIR}/${START_UP} @ONLY)
+f(WIN32)
+message("win-startup 1: ${OUTDIR}/${GRASS_INSTALL_ETCDIR}/${START_UP}")
+  configure_file(grass.py ${OUTDIR}/${GRASS_INSTALL_ETCDIR}/${START_UP} @ONLY)
+else()
+  configure_file(grass.py ${OUTDIR}/${CMAKE_INSTALL_BINDIR}/${START_UP} @ONLY)
+endif()
 
 set(GRASS_PYDIR ${GRASS_INSTALL_PYDIR_PATH})
+message("win-startup 2: ${CMAKE_CURRENT_BINARY_DIR}/${START_UP}")
 configure_file(grass.py ${CMAKE_CURRENT_BINARY_DIR}/${START_UP} @ONLY)
 unset(GRASS_PYDIR)
 
+message("win-startup 3: ${CMAKE_INSTALL_BINDIR}")
 install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/${START_UP}
         DESTINATION ${CMAKE_INSTALL_BINDIR})
 
 file(TO_NATIVE_PATH "${OUTDIR}/${GISBASE_DIR}" gisbase_init_dir)
 
+if(WIN32)
+  set(GRASS_PYTHON ${PYTHON_EXECUTABLE})
+  configure_file(${script_input_file_name} ${OUTDIR}/${CMAKE_INSTALL_BINDIR}/${script_file_name} @ONLY)
+endif()
+
 file(TO_NATIVE_PATH ${OUTDIR}/${GRASS_INSTALL_BINDIR} grass_dll_dir)
 set(DLL_PATH_LIST)
 list(APPEND DLL_PATH_LIST ${grass_dll_dir})
