--- lib/init/grass.bat.in.orig	2025-07-18 06:56:39.193870057 -0600
+++ lib/init/grass.bat.in	2026-04-28 16:48:03.762242268 -0600
@@ -11,14 +11,14 @@
 
 ::set FONTCONFIG_FILE=%GISBASE%\etc\fonts.conf
 
-set PATH=@gisbase_init_dir@\bin;@BINARY_DIR@;@DLL_PATH_LIST@;%PATH%
+set GISBASE=%~dp0\..\@gisbase_init_dir@
 
-set GISBASE=@gisbase_init_dir@
+set PATH=%GISBASE%\bin;@BINARY_DIR@;@DLL_PATH_LIST@;%PATH%
 
-set PYTHONPATH=@gisbase_init_dir@\etc\python;@gisbase_init_dir@\gui\wxpython;@gisbase_init_dir@\etc
+set PYTHONPATH=%GISBASE%\etc\python;%GISBASE%\gui\wxpython;%GISBASE%\etc
 
 set GRASS_PYTHON=@GRASS_PYTHON@
 
 set GRASS_CONFIG_DIR=@grass_config_dir@
 
-"@GRASS_PYTHON@" "@gisbase_init_dir@\etc\@START_UP@" %*
+"@GRASS_PYTHON@" "%GISBASE%\etc\@START_UP@" %*
