--- cmake/modules/build_script_in_subdir.cmake.orig	2026-04-27 18:39:39.717681210 -0600
+++ cmake/modules/build_script_in_subdir.cmake	2026-04-27 19:22:03.277658852 -0600
@@ -15,24 +15,28 @@
     ""
     ${ARGN})
 
+  message("===== locale 1: ${G_DEST_DIR} =====")
   if(NOT G_DEST_DIR)
     set(G_DEST_DIR ${GRASS_INSTALL_SCRIPTDIR})
   endif()
 
   set(G_NAME ${dir_name})
 
+  message("===== locale 2: ${G_NAME} =====")
   if(G_PLAIN_PY)
     set(G_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR})
   else()
     set(G_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/${G_NAME})
   endif()
 
+  message("===== locale 3: ${G_SRC_DIR} =====")
   file(GLOB PYTHON_FILES "${G_SRC_DIR}/*.py")
   if(NOT PYTHON_FILES)
     message(FATAL_ERROR "[${G_NAME}]: No PYTHON_FILES found.")
   endif()
 
   set(SRC_SCRIPT_FILE ${G_SRC_DIR}/${G_NAME}.py)
+  message("===== locale 4: ${SRC_SCRIPT_FILE} =====")
 
   if(NOT EXISTS ${SRC_SCRIPT_FILE})
     message(FATAL_ERROR "${SRC_SCRIPT_FILE} does not exists")
@@ -45,6 +49,7 @@
   endif()
 
   set(HTML_FILE ${G_SRC_DIR}/${G_NAME}.html)
+  message("===== locale 5: ${HTML_FILE} =====")
 
   configure_file(
     ${G_SRC_DIR}/${G_NAME}.py
@@ -64,6 +69,7 @@
   set(TRANSLATE_C_FILE
       ${CMAKE_SOURCE_DIR}/locale/scriptstrings/${G_NAME}_to_translate.c)
 
+  message("===== locale 6: ${CMAKE_SOURCE_DIR}...${TRANSLATE_C_FILE} =====")
   add_custom_command(
     OUTPUT ${TRANSLATE_C_FILE}
     COMMAND
