--- CMakeLists.txt.orig	2026-04-26 21:10:50.698117731 -0600
+++ CMakeLists.txt	2026-04-27 21:54:21.788820688 -0600
@@ -284,16 +284,17 @@
 foreach(d ${ALL_SUBDIRS})
   add_subdirectory(${d})
 endforeach()
+
+if(WITH_GUI)
+  add_subdirectory(gui)
+endif()
+
 add_custom_target(
   ALL_MODULES
   DEPENDS ${modules_list}
   COMMENT "Building all modules.")
 set_target_properties(ALL_MODULES PROPERTIES FOLDER Misc)
 
-if(WITH_GUI)
-  add_subdirectory(gui)
-endif()
-
 if(WITH_DOCS)
   add_subdirectory(doc)
   add_subdirectory(man)
