--- cmake/locale_strings.cmake.orig	2025-08-08 11:03:34.066498790 -0600
+++ cmake/locale_strings.cmake	2026-04-27 18:43:53.459680642 -0600
@@ -56,6 +56,7 @@
 
 if(WIN32)
   set(PGM_NAME ${G_NAME})
+  message("===== locale: ${SOURCE_DIR}/cmake/windows_launch.bat.in....${SCRIPTDIR}/${G_NAME}.bat =====")
   configure_file(${SOURCE_DIR}/cmake/windows_launch.bat.in
                  ${SCRIPTDIR}/${G_NAME}.bat @ONLY)
 endif(WIN32)
